pom.xml

```diff
@@ -229,6 +230,45 @@
 		    <module>bundles/org.eclipse.osgi.tests</module>
         </modules>
     </profile>
+    <profile>
+      <id>fetch-natives</id>
```

Comment:
Actually I would have liked to add this to the launcher-binary-parent, but because it's only the parent of all `org.eclipse.equinox.launcher.*` artifact the defined antrun plugin to clone the binaries-repo has a high chance to run concurrently which will probably lead to build failures du to concurrent clones of the repository.

Adding it to the root parent ensures that there is only exactly one execution of the clone.

Reply:
Unfortunately I'm not aware of a convenient trick to just execute the mojo on the reactor root, which is why the execution runs on each project (if the profile is active) and just skips the cloning.